CONTRIBUTING.md

```diff
@@ -104,9 +104,9 @@ Crayons repository uses github pre commit hooks, follow the below steps to commi
 
 8. Give a commit message when it prompts.
 
-9. Run `git push -u origin <your-branch>`, to push your changes to the forked repo. git hooks will run all the test before pushing the changes. if any test fails, rewrite the tests and make sure all the test passes before pushing again.
+9. Run `git push -u origin <your-branch>`, to push your changes to the forked repo. git hooks will run all the test before pushing the changes. if any test fails, rewrite the tests and make sure all the tests pass before pushing again.
```

Comment:
Could we modify this as follows?
Run `git push -u origin <your-branch>`, to push your changes to the forked repo. git hooks run all required tests before pushing the changes. If any test fails, rewrite the test and ensure that all the tests pass before pushing again.